sdk/src/types/block/output/nft.rs

```diff
@@ -263,14 +263,15 @@ impl NftOutputBuilder {
     }
 
     ///
-    pub fn finish(self) -> Result<NftOutput, Error> {
+    pub fn finish(self) -> Result<NftOutput, OutputError> {
         let unlock_conditions = UnlockConditions::from_set(self.unlock_conditions)?;
 
         verify_unlock_conditions(&unlock_conditions, &self.nft_id)?;
 
         let features = Features::from_set(self.features)?;
 
-        verify_restricted_addresses(&unlock_conditions, NftOutput::KIND, features.native_token(), self.mana)?;
+        verify_restricted_addresses(&unlock_conditions, NftOutput::KIND, features.native_token(), self.mana)
+            .map_err(UnlockConditionError::from)?;
```

Comment:
missing from impl?

Reply:
Nope, this one is on purpose because we need to specify that this address error is from an unlock condition